packages/components/index.d.ts

```diff
@@ -398,13 +398,13 @@ interface InpuBasicProps {
   ref?: React.Ref<HTMLElement>;
 }
 
-interface InputProps extends InpuBasicProps {
+interface InputProps extends InputBasicProps {
   onChange?: React.ChangeEventHandler<HTMLInputElement>;
 }
 
-interface TextFieldProps extends InpuBasicProps {
-  onChange?: (e: Event, valid: boolean) => void;
-  onBlur?: (e: Event) => void;
+interface TextFieldProps extends InputBasicProps {
+  onChange?: (e: React.ChangeEvent<HTMLTextAreaElement> | React.ChangeEvent<HTMLInputElement>, valid: boolean) => void;
```

Comment:
Some [reference](https://github.com/mui-org/material-ui/blob/next/packages/material-ui/src/InputBase/InputBase.d.ts#L142) for simplifying `onChange` event type